LambdaS3FileZipper.Test/LambdaS3FileZipper.Test.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <LangVersion>7.1</LangVersion>
```

Comment:
Have we verified this won't cause a problem with Lambda?

Reply:
Yes, I've already tested it in **Lambda**: given that it is still compiled for the same framework, it doesn't really affect things.